src/niweb/niweb/settings/common.py

```diff
@@ -190,8 +199,8 @@
 
 
 ### LOGIN conf
-DJANGO_LOGIN_DISABLED = environ.get('DJANGO_LOGIN_DISABLED', False)
-SAML_ENABLED = environ.get('SAML_ENABLED', False)
+DJANGO_LOGIN_DISABLED = environ.get('DJANGO_LOGIN_DISABLED', 'False').lower() == 'false'
+SAML_ENABLED = environ.get('SAML_ENABLED', 'False').lower() == 'false'
```

Comment:
This is still wrong... If i set `DJANGO_LOGIN_DISABLED` to `True` it will be set to `False` with this.
I can fix it myself though.